### PR TITLE
Fix path reference and use a single global helpers variable

### PR DESCRIPTION
**Pull request by chrispbailey**
I had an issue where the path to the file was wrong and just needed a path.delimiter character.

In trying to fix this I created my own package name so turned that into a variable and also copied some dependency loading functionality from another linter.

### CODE REVIEW

File: lib/main.js [hunk 1]
```diff
@@ -167,7 +184,7 @@ export default {
                 location: {
                   // ansible-lint plus atom-linter combined now lose pathing info for this kind of parsing
                   // prepend dir to file
-                  file: dir + lint_matches[1],
+                  file: dir + path.delimiter + lint_matches[1],
```
Comment (mschuchard) [inline lib/main.js:187]: This does need a character to separate the filename from the path, like another PR to address the same issue my Terraform package had with the same workaround for the atom-linter relative pathing bug also introduced. According to [documentation](https://nodejs.org/api/path.html#path_path_delimiter), this is not the necessary fix though. Ideally atom-linter would fix its relative pathing bug instead, but that does not seem to be a possibility.

Comment (chrispbailey) [reply lib/main.js:187]: Maybe it works because it doesn't need the `dir` at all - so only `lint_matches[1]` is required?

Comment (mschuchard) [reply lib/main.js:187]: For the same workaround in the Terraform package, a `/` was concatenated between the `dir` and `file`. I checked and apparently `path` interprets a `/` correctly independent of platform; as weird as that sounds.

**Comment by mschuchard** [comment]
Can you redo the PR with bfb2f9563bbe854c63f4a1dcd026c87f915b5f7c? That would fix the issue, whereas the other commit with a global "find-and-replace" on the package name is extraneous.
Thanks!

**Comment by mschuchard** [comment]
Superseded by #40.

**Pull request closed without merging**